stac/catalog.json

```diff
@@ -28,7 +28,7 @@
     {
       "href": "./spacenet7/catalog.json",
       "rel": "child",
-      "title": "Spacenet 7 Data & Labels",
+      "title": "Spacenet 7 Data \u0026 Labels",
```

Comment:
what's with this? I think @m-mohr might have just changed them one way?

Reply:
A difference in the JSON writers. I mangled them through the JS parser and it uses the "ASCII" chars, while Go(?) seems to use/write this unicode thing. It was like that before in the other files, too. Shouldn't be an issue though, just looks a bit weird if you look at the JSON files.

Reply:
It is not an issue.  Many encoders escape ampersand to avoid downstream html injection.